bin/sync_cms_to_repo.sh

```diff
@@ -0,0 +1,20 @@
+PR_TITLE="chore: cms to repo sync"
+BRANCH_NAME="sync_cms_to_plugins_$(date +%s)"
+
+git branch $BRANCH_NAME
+git switch $BRANCH_NAME
+
+echo "Syncing CMS to plugins"
+npx tsx bin/sync_cms_to_plugins_repo.js
+
+echo "Creating PR \"$PR_TITLE\" for branch $BRANCH_NAME"
+
+// This is the only file we want to commit
+git add site/plugins.json
+
+// See if we have any changes. We should.
+if [[ -n "$(git status --porcelain)" ]]; then
+  git commit -m $PR_TITLE
+  git push origin $BRANCH_NAME
+  gh pr create --title $PR_TITLE --body "This is an automated PR to sync the CMS to the repo" --label "cms_sync" --label "automerge"
```

Comment:
I created a label in the repo called `cms_sync` for filtering of automated PRs, and I also add the `automerge` label which should auto merge the PR that gets generated.

Reply:
noice